AmazonChimeSDK/AmazonChimeSDKTests/internal/audio/DefaultAudioClientObserverTests.swift

```diff
@@ -193,6 +193,42 @@ class DefaultAudioClientObserverTests: XCTestCase {
         wait(for: [expect], timeout: defaultTimeout)
     }
 
+    func testAudioClientStateChanged_NotifiesOfInputDeviceFailure() {
```

Comment:
I feel like this could be a more specific name incase there are other reasons a device can fail. However, this is minor.